src/components/modebar/modebar.js

```diff
@@ -172,8 +172,12 @@ proto.createIcon = function(thisIcon) {
     icon.setAttribute('width', (thisIcon.width / iconHeight) + 'em');
     icon.setAttribute('viewBox', [0, 0, thisIcon.width, iconHeight].join(' '));
 
+    var transform = name === 'toggleSpikelines' ?
```

Comment:
Hmm. I did break something, the new button doesn't seem to work:

![peek 2017-04-07 17-28](https://cloud.githubusercontent.com/assets/6675409/24820565/bce06820-1bb7-11e7-97bd-bfdbdd7cec6e.gif)